src/Nethermind/Nethermind.Trie/INodeStorage.cs

```diff
@@ -18,14 +18,14 @@ public interface INodeStorage
     /// </summary>
     public bool RequirePath { get; }
 
-    byte[]? Get(Hash256? address, in TreePath path, in ValueHash256 keccak, ReadFlags readFlags = ReadFlags.None);
-    void Set(Hash256? address, in TreePath path, in ValueHash256 hash, ReadOnlySpan<byte> data, WriteFlags writeFlags = WriteFlags.None);
+    byte[]? Get(in ValueHash256? address, in TreePath path, in ValueHash256 keccak, ReadFlags readFlags = ReadFlags.None);
```

Comment:
Address should be kept a reference type as the triestore should have it as `Hash256` so that it can be deduplicated across keys of the same storage.

Reply:
You are right those were unnecessary